docs/user_guide/vis/index.rst

```diff
@@ -1,88 +1,285 @@
+Starting and stopping the IFV is fast and does not consume many resources, a fact which we hope encourages
+its frequent use.
+
+Quickstart
+----------
+This section will walk you through constructing a basic flowsheet and visualizing it with the IFV.
+
+Before starting to use the IFV, you must have installed IDAES. For detailed instructions on how to do this,
+see :ref:`IDAES Installation`.
+
+In this quickstart we will use a Jupyter Notebook to compose our flowsheet and view it with the IFV. So, first
+you need to open a new, blank, Jupyter Notebook. You can start the Jupyter Notebook (server) from the terminal command-line with
+the command ``jupyter notebook``. See the `official Jupyter Notebook documentation <https://jupyter-notebook.readthedocs.io/>`_
+for detailed instructions. The Jupyter Notebook is a web application, so you should get a web page (often as a tab
+on an open browser window) that shows the contents of the directory from where you issued the command. Navigate to
+a directory where you want to save your work, then click on the "New" button in the upper-right corner to create
+a new notebook. You may have multiple options under the "Python" sub-menu; make sure you select a Python installation
+that has IDAES already installed.
+
+We need to create, in this Jupyter Notebook, an IDAES flowsheet to visualize. Paste the following code (taken from the Flash unit tutorial)
+into the first cell in the notebook::
+
+    from pyomo.environ import ConcreteModel, SolverFactory, Constraint, value
+    from idaes.core import FlowsheetBlock
+    from idaes.generic_models.properties.activity_coeff_models.BTX_activity_coeff_VLE \
+        import BTXParameterBlock
+    from idaes.generic_models.unit_models import Flash
+    # Model and flowsheet
+    m = ConcreteModel()
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+    # Flash properties
+    m.fs.properties = BTXParameterBlock(default={"valid_phase": ('Liq', 'Vap'),
+                                                "activity_coeff_model": "Ideal",
+                                                "state_vars": "FTPz"})
+    # Flash unit
+    m.fs.flash = Flash(default={"property_package": m.fs.properties})
+    m.fs.flash.inlet.flow_mol.fix(1)
+    m.fs.flash.inlet.temperature.fix(368)
+    m.fs.flash.inlet.pressure.fix(101325)
+    m.fs.flash.inlet.mole_frac_comp[0, "benzene"].fix(0.5)
+    m.fs.flash.inlet.mole_frac_comp[0, "toluene"].fix(0.5)
+    m.fs.flash.heat_duty.fix(0)
+    m.fs.flash.deltaP.fix(0)
+    # Initialize
+    m.fs.flash.initialize()
+
+
+Run that cell (hit shift-enter), and you should see some output from the last call to ``initialize()``, ending with something like ::
+
+    2020-11-24 08:41:53 [INFO] idaes.init.fs.flash: Initialization Complete: optimal - Optimal Solution Found
+
+This means the initialization succeeded, and you can move on.
+
+.. warning::
```

Comment:
not sure if this warning is needed
there are many reasons your flowhseet failed to build/run
what about a note:: pointing to the idaes workshop to learn to build, initialize, and run IDAES flowsheets?

Reply:
I would prefer to keep this self-contained. We will no doubt move those things around. I can add a link within the docs though and Sphinx will check that this is valid.

Reply:
I changed warning to note but I think this type of error is relatively easy to happen and would be frustrating to most users. If we have documentation of common errors in another place I can move it there (but afaik that's not the case). This is much more about environment setup than using IDAES.